deployment/minikube/generate_yaml.sh

```diff
@@ -38,5 +38,6 @@ yq w -i --style=double --inplace -- deployment.yaml spec.template.spec.container
 
 # ESP service configuration
 yq w --style=double ../template/endpoints.yaml.tmpl name $SERVICE_NAME > endpoints.yaml
+yq w -i endpoints.yaml title $service_name
```

Comment:
should this be all caps as well, like in previous line?
$SERVICE_NAME

Reply:
updated.